com.kjxbyz.PicGuard.yml

```diff
@@ -0,0 +1,70 @@
+id: com.kjxbyz.PicGuard
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
```

Comment:
Does this actually need gnome runtime? Or freedesktop will work too?

Reply:
I developed it using flutter, and I'm not sure if I have to use the `gnome` runtime, but I think it should work fine on `freedesktop`. How can I support both runtimes at the same time?

Reply:
Can you change it to freedesktop? if it builds and runs, then it is fine to use that.

GNOME is a superset of the freedesktop runtime, you don't need to support both.

Reply:
Then I need to change to freedesktop and test it locally.